ingestion/tests/unit/topology/metadata/test_atlas.py

```diff
@@ -0,0 +1,506 @@
+    OpenMetadataConnection,
+)
+from metadata.generated.schema.entity.services.databaseService import (
+    DatabaseConnection,
+    DatabaseService,
+)
+from metadata.generated.schema.metadataIngestion.workflow import (
+    OpenMetadataWorkflowConfig,
+)
+from metadata.generated.schema.type.basic import Href
+from metadata.generated.schema.type.entityReference import EntityReference
+from metadata.generated.schema.type.tagLabel import TagLabel
+from metadata.ingestion.ometa.ometa_api import OpenMetadata
+from metadata.ingestion.source.metadata.atlas.client import AtlasClient
+from metadata.ingestion.source.metadata.atlas.metadata import AtlasSource
+
+mock_atlas_config = {
+    "source": {
+        "type": "Atlas",
+        "serviceName": "local_atlas",
+        "serviceConnection": {
+            "config": {
+                "type": "Atlas",
+                "hostPort": "http://192.168.1.8:21000",
+                "username": "username",
+                "password": "password",
+                "databaseServiceName": ["hive"],
+                "messagingServiceName": [],
+            }
+        },
+        "sourceConfig": {"config": {"type": "DatabaseMetadata"}},
+    },
+    "sink": {"type": "metadata-rest", "config": {}},
+    "workflowConfig": {
+        "openMetadataServerConfig": {
+            "hostPort": "http://localhost:8585/api",
+            "authProvider": "openmetadata",
+            "securityConfig": {
+                "jwtToken": "eyJraWQiOiJHYjM4OWEtOWY3Ni1nZGpzLWE5MmotMDI0MmJrOTQzNTYiLCJ0eXAiOiJKV1QiLCJhbGc"
+                "iOiJSUzI1NiJ9.eyJzdWIiOiJhZG1pbiIsImlzQm90IjpmYWxzZSwiaXNzIjoib3Blbi1tZXRhZGF0YS5vcmciLCJpYXQiOjE"
+                "2NjM5Mzg0NjIsImVtYWlsIjoiYWRtaW5Ab3Blbm1ldGFkYXRhLm9yZyJ9.tS8um_5DKu7HgzGBzS1VTA5uUjKWOCU0B_j08WXB"
+                "iEC0mr0zNREkqVfwFDD-d24HlNEbrqioLsBuFRiwIWKc1m_ZlVQbG7P36RUxhuv2vbSp80FKyNM-Tj93FDzq91jsyNmsQhyNv_fN"
+                "r3TXfzzSPjHt8Go0FMMP66weoKMgW2PbXlhVKwEuXUHyakLLzewm9UMeQaEiRzhiTMU3UkLXcKbYEJJvfNFcLwSl9W8JCO_l0Yj3u"
+                "d-qt_nQYEZwqW6u5nfdQllN133iikV4fM5QZsMCnm8Rq1mvLR0y9bmJiD7fwM1tmJ791TUWqmKaTnP49U493VanKpUAfzIiOiIbhg"
+            },
+        }
+    },
+}
+
+
+mock_file_path = (
+    Path(__file__).parent.parent.parent / "resources/datasets/atlas_dataset.json"
+)
+with open(mock_file_path, encoding="UTF-8") as file:
+    mock_data: dict = json.load(file)
+
+
+LIST_ENTITIES = [
+    "b233b2ae-8a4a-44a3-b446-4027462b2cc6",
+    "eff274ef-f84d-4c58-81d6-c309663b887d",
+    "1697e9bc-2fb0-4d11-8294-7e0da9c74830",
+    "aabdc6c0-96c1-451c-98b5-7a7e48c319be",
+]
+
+
+def mock_get_entity(self, table):  # pylint: disable=unused-argument
+    return mock_data
+
+
+def mock_list_entities(self, entity_type):  # pylint: disable=unused-argument
+    return LIST_ENTITIES
+
+
+EXPECTED_DATABASE_DESCRIPTION = "THIS IS TEST_DESCRIPTION FOR DATABASE"
+
+EXPTECTED_DATABASE_SCHEMA_DESCRIPTION = "THIS IS TEST_DESCRIPTION FOR DATABASE"
```

Comment:
```suggestion
EXPTECTED_DATABASE_SCHEMA_DESCRIPTION = "THIS IS TEST_DESCRIPTION FOR DATABASE SCHEMA"
```

Reply:
So It ingest same for database and schema . So value for database and schema are same.